File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        map<int, int> dict;
        for(int i : nums)
        {
            dict[i]++;
            if (dict[i] > 1)
                return true;
        }
        
        return false;
        
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.cs

public class Solution {
    public bool ContainsDuplicate(int[] nums)
    {
        HashSet<int> hash = new HashSet<int>();
        foreach (int i in nums)
        {
            if (hash.Contains(i))
            {
                return true;
            }
            hash.Add(i);
        }
        return false;
    }
}